projectns/cs_claim.c

```diff
@@ -25,6 +26,13 @@ static void cmd_claim(sourceinfo_t *si, int parc, char *parv[])
 		return;
 	}
 
+	// make sure the channel exists
+	if (!(c = channel_find(name)))
```

Comment:
```suggestion
	if (!channel_find(name))
```

we can skip naming c if we're never going to use it again